src/components/Button/Button.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { describe, test } from "vitest";
+import { render } from "@testing-library/react";
+import { Button } from "./Button";
+
+describe("<Button />", () => {
+  test("should render", () => {
+    render(<Button variant="primary">test</Button>);
+  });
```

Comment:
Should add tests to the differents variants of buttons, size of buttons, buttons with icons, etc.